### DISCUSSION TITLE
Align similar lines

### DISCUSSION THREAD

**Discussion by Criscia68** [WinMerge/winmerge#2541] [Ideas]
The “Align similar lines” function does not work correctly  especially with three files.
Is it possible to do this manually via mouse-activated marks?

![image](https://github.com/user-attachments/assets/dff9d66d-b8f8-4c49-b5de-135fe7feb602)

**Comment by sdottaka** [comment]
WinMerge has a function that allows you to add a Synchronization Point.

For example, if you want the following lines in the attached image to be displayed in the same vertical position, move to the relevant lines in each pane and select Merge → Add Synchronization Point menu item. A Synchronization Point will be added and these lines will be displayed in the same vertical position.

Left:
    CM  Arm_to_ccw_0_cw_180               ; 54 (gripper turn to maga
Middle:
    CM  T_arm_to_ccw_0_cw_180             ; 48 (gripper turn to maga
Right:
    CM  T_arm_to_ccw_0_cw_180             ; 54 (gripper turn to maga


![image](https://github.com/user-attachments/assets/a0709b11-1061-475a-a15f-41605af52c13)

**Reply by Criscia68** [reply]
What if I want to delete only one sync point?

![image](https://github.com/user-attachments/assets/6d86c828-d728-4701-ac1e-b95061a2846b)

**Reply by sdottaka** [reply]
Unfortunately there is no menu item to remove a single synchronization point.
However, you can delete a synchronization point by overwriting an existing one as follows:
![removesyncpoint2](https://github.com/user-attachments/assets/b9e7ef59-48e8-41e1-86de-5ea5ab56141a)
